cert/aziot-certd/src/config.rs

```diff
@@ -3,19 +3,21 @@
 #[derive(Debug, PartialEq, serde::Deserialize)]
 pub struct Config {
 	/// Path of home directory.
-	pub(crate) homedir_path: std::path::PathBuf,
+	pub homedir_path: std::path::PathBuf,
 
 	/// Configuration of how new certificates should be issued.
-	pub(crate) cert_issuance: CertIssuance,
+	pub cert_issuance: CertIssuance,
 
 	/// Map of preloaded certs from their ID to their location.
 	#[serde(default)]
-	pub(crate) preloaded_certs: std::collections::BTreeMap<String, PreloadedCert>,
+	pub preloaded_certs: std::collections::BTreeMap<String, PreloadedCert>,
+
+	pub endpoints: Endpoints,
```

Comment:
Missing doc comment? (Same in other config.rs files)

Reply:
Fixed.